css/main.css

```diff
@@ -0,0 +1,353 @@
+}
+
+.bg-3 {
+    background-color: var(--third-bg-color);
+}
+
+/* Navigation */
+.navbar {
+    background: var(--navbar-color);
+}
+
+.nav-link,
+.navbar-brand {
+    font-size: 1.8rem;
+    cursor: pointer;
+}
+
+.nav-link {
+    margin-right: 1rem;
+}
+
+.nav-link:hover {
+    background: rgba(0, 0, 0, 0);
+    color: var(--title-txt-color);
+}
+
+.navbar-collapse {
+    justify-content: flex-end;
+}
+
+.navbar-toggler {
+    background: var(--navbar-color);
+}
+
+/*  Hero */
+
+.header {
+    background-image: url("../images/bkgtop.png");
+    background-attachment: fixed;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 35rem;
+}
+
+.overlay {
+    position: absolute;
+    height: 35rem;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-color: rgb(255, 0, 0, 0.1);
+}
+
+.title-position {
+    position: absolute;
+    width: 100%;
+    top: 8rem;
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.hero-title {
+    font-size: 3rem;
+    line-height: 1.5;
+    text-align: center;
+    -webkit-box-sizing: content-box;
+    -moz-box-sizing: content-box;
+    box-sizing: content-box;
+    padding: 2rem;
+    border: none;
+    font-family: 'Baloo 2', cursive;
+    color: rgba(255, 255, 255, 1);
+    -o-text-overflow: ellipsis;
+    text-overflow: ellipsis;
+    letter-spacing: 0.5rem;
+    text-shadow: 0 0 10px rgb(255, 255, 255), 0 0 20px rgb(255, 255, 255),
+        0 0 30px rgb(255, 255, 255), 0 0 40px rgba(255, 17, 17, 1),
+        0 0 70px rgba(255, 17, 17, 1), 0 0 80px rgba(255, 17, 17, 1),
+        0 0 100px rgba(255, 17, 17, 1), 0 0 150px rgba(255, 17, 17, 1);
+    -webkit-transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1) 10ms;
+    -moz-transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1) 10ms;
+    -o-transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1) 10ms;
+    transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1) 10ms;
+
+}
+
+.hero-title:hover,
+.hero-title:focus {
+    letter-spacing: 8px;
+    text-shadow: 0 0 5px rgb(255, 255, 255), 0 0 10px rgb(255, 255, 255),
+        0 0 15px rgb(255, 255, 255), 0 0 20px rgba(247, 142, 4, 1),
+        0 0 35px rgba(247, 142, 4, 1), 0 0 40px rgba(247, 142, 4, 1),
+        0 0 50px rgba(247, 142, 4, 1), 0 0 75px rgba(247, 142, 4, 1);
+}
+
+
+/* Community Section*/
+
+.community-sec {
+    padding-bottom: 5rem;
+}
+
+.text {
+    padding-top: 5rem;
+    padding-bottom: 5rem;
+}
+
+.buttons {
+    margin-top: 5rem;
+    display: flex;
+    flex-direction: column;
+}
+
+/*Kizomba Section*/
+.kizomba-sec {
+
+    padding-bottom: 3rem;
+    padding-top: 0.3px;
+}
+
+.img {
+    margin: auto !important;
+    width: 300px;
+    height: auto;
+    display: block;
+    border-radius: 0.3px;
+}
+
+/*Buttons*/
+
+.btn-link {
+    border-radius: 20px;
+    color: #af0404;
+    font-family: Open Sans;
+    font-size: 26px;
+    font-weight: 100;
+    padding: 20px;
+    background-color: white;
+    opacity: 0.75;
```

Comment:
The button doesn't get fully white, as you want, because it has an opacity. You need to remove the opacity. If you want opacity in other property (for example box-shadow), you'll need to use `rgba()` colors.